src/scores/categorical/contingency_impl.py

```diff
@@ -887,11 +889,10 @@ def symmetric_extremal_dependence_index(self) -> xr.DataArray:
 
 
         References:
-            Ferro, C.A. and Stephenson, D.B., 2011. Extremal dependence indices: Improved
-            verification measures for deterministic forecasts of rare binary events. Weather
-            and Forecasting, 26(5), pp.699-713. https://doi.org/10.1175/WAF-D-10-05030.1
+            Ferro, C. A. T., and D. B. Stephenson, 2011: Extremal Dependence Indices:
+            Improved Verification Measures for Deterministic Forecasts of Rare Binary
+            Events. Wea. Forecasting, 26, 699–713, https://doi.org/10.1175/WAF-D-10-05030.1.
```

Comment:
Thanks for adding the "T" in "Ferro, C. A. T.".

Personally, I liked that the previous version included the full journal title and included the issue number. But that's a subjective opinion.